pkg/visor/init.go

```diff
@@ -181,10 +259,11 @@ func initTransport(v *Visor) bool {
 		DiscoveryClient: tpdC,
 		LogStore:        logS,
 	}
-
-	tpM, err := transport.NewManager(v.MasterLogger().PackageLogger("transport_manager"), v.net, &tpMConf)
+	managerLogger := v.MasterLogger().PackageLogger("transport_manager")
+	tpM, err := transport.NewManager(managerLogger, v.net, &tpMConf)
```

Comment:
Does any reason for this change? [a personal question]

Reply:
No, I don't think there is, maybe it's a bit shorter and looks better. Could be that there were more changes that required logger variable but later I removed them.